PROJECT

```diff
@@ -5,3 +5,6 @@ resources:
 - group: infrastructure
   version: v1alpha1
   kind: DockerMachine
+- group: infrastructure
+  version: v1alpha1
```

Comment:
Shouldn't this be `v1alpha2`?

@chuckha same for the above DockerMachine.

Reply:
oh. Are we matching versions? These didn't exist for v1alpha1, this is the very first version of these types.